example_dags/example_sqlite_load_transform.py

```diff
@@ -33,7 +33,7 @@ def get_top_five_animations(input_table: Table):
 
     imdb_movies = aql.load_file(
         input_file=File(
-            path="https://raw.githubusercontent.com/astronomer/astro-sdk/main/tests/data/imdb.csv"
+            path="https://raw.githubusercontent.com/astronomer/astro-sdk/main/tests/data/imdb_v2.csv"
```

Comment:
Github URLs in past have thrown 500 errors which have made the tests flaky. This can changed to S3 URLs

Reply:
@sunank200 Do we have a public S3 bucket already? If we do, I can change it - otherwise, could we do that as part of a separate PR, WDYT?